### PR TITLE
@ #40 | should update to the new location sync format

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #40 
please check this PR! Thank you

### CODE REVIEW

**Comment by hoatle** [comment]
@hieptranquoc make sure this works properly https://github.com/teracyhq-incubator/teracy-dev-k8s/blob/develop/lib/teracy-dev-k8s/processors/settings.rb#L51

**Comment by hieptranquoc** [comment]
I've updated, please check this PR again. Thank you!

File: config.yaml [hunk 1]
```diff
@@ -2,10 +2,13 @@
 teracy-dev-k8s:
   # configurable here
   kubespray:
-    lookup_path: extensions # relative to the Vagrantfile
```
Comment (hoatle) [inline config.yaml]: I prefer keeping this as is:
- the input matches the location input of Gitsynch::sync
- do not break compatibility
- no reason to change

**Comment by hieptranquoc** [comment]
@hoatle I've updated! Please check this PR again.

**Pull request merged**